pkg/flow/flow_services_test.go

```diff
@@ -170,6 +170,76 @@ func TestFlow_GetServiceConsumers(t *testing.T) {
 	require.Equal(t, []any{svcB}, consumers)
 }
 
+func TestFlow_GetServiceConsumers_Modules(t *testing.T) {
```

Comment:
Do we need to test more complicated situations too - e.g. a module A, which imports a module B, where module B is the one using the service?

Reply:
We could, but I'm not sure it's necessary. There's not much difference in the code between one layer of modules and many layers of modules.